#! /bin/sh /usr/share/dpatch/dpatch-run
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: this patch is used to fix inbound pathes, no configure switch.

@DPATCH@
--- fidogate/build/config.make.in.orig	2004-03-20 19:42:58.000000000 +0200
+++ fidogate/build/config.make.in	2004-03-20 19:43:15.000000000 +0200
@@ -66,8 +66,8 @@
 DEFAULT_V_SPOOLDIR	= @SPOOLDIR@
 DEFAULT_V_BTBASEDIR	= @btbasedir@
 DEFAULT_V_NETMAILDIR	= @netmaildir@
-DEFAULT_V_INBOUND	= $(DEFAULT_V_BTBASEDIR)/in
-DEFAULT_V_PINBOUND	= $(DEFAULT_V_BTBASEDIR)/pin
+DEFAULT_V_INBOUND	= $(DEFAULT_V_BTBASEDIR)/inb
+DEFAULT_V_PINBOUND	= $(DEFAULT_V_BTBASEDIR)/inb
 DEFAULT_V_UUINBOUND     = $(DEFAULT_V_BTBASEDIR)/uuin
 DEFAULT_V_FTPINBOUND    = $(DEFAULT_V_BTBASEDIR)/ftpin
 
